.github/workflows/copyright.yaml

```
@@ -0,0 +1,22 @@
+name: Update copyright year(s) in license file
+
+on:
+  schedule:
+    - cron: '0 3 1 1 *' # 03:00 AM on January 1
+
+jobs:
+  update-license-year:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: >
+            *.rs
+            *.proto
```

:wrench: looking at the docs, https://github.com/FantasticFiasco/action-update-license-year#i-want-to-update-all-my-licenses-i-have-more-than-one, it says to use the literal style `|`, but this is using the folded style `>`
Also looking at some yaml guidance it seems that `*` not surrounded by spaces should be quoted.
```suggestion
          path: |
            '*.rs'
            '*.proto'
```